src/lib/ts/analyse-dependencies-transformer.ts

```diff
@@ -8,12 +8,13 @@ export const analyseDependencies = (analyser: DependencyAnalyser) => (context: t
   sourceFile: ts.SourceFile
 ): ts.SourceFile => {
   // skip source files from 'node_modules' directory (third-party source)
-  if (sourceFile.fileName.includes('node_modules')) {
+  // 'ngfactory' and 'ngstyles' should also be skipped
+  if (/node_modules|\.ngfactory|\.ngstyle/.test(sourceFile.fileName)) {
```

Comment:
FYI: The fix is to skip `.ngfactory` and `.ngstyle` files from dependency analyser